notebooks/QA_extraction_generation.ipynb

```diff
@@ -0,0 +1,556 @@
+{
```

Comment:
<p>Pretty cool!</p><p>Makes me think, the variables that we need to experiment with here could be : context, temperature, max-length, num_beams, but also maybe 'repetition_penalty' 'length_penalty' etc. Maybe parameters <a href="https://huggingface.co/docs/transformers/main_classes/text_generation#transformers.GenerationConfig.length_penalty" target="_blank">here</a> would be worth a try.</p><p>The major one to fix would be context because in extractive and open book generative it will be critical.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/foundation-models-for-documentation/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

Reply:
<p>Thank you Shrey. Yes, there are room for looking into these parameters. </p><p> </p>